new-integration-tests/src/test/java/oracle/weblogic/kubernetes/ItSimpleOperatorValidation.java

```diff
@@ -56,12 +58,16 @@
 @ExtendWith(Timing.class)
 // by implementing the LoggedTest, we will automatically get a logger injected and it
 // will also automatically log entry/exit messages for each test method.
+@ExtendWith(IntegrationTestWatcher.class)
 class ItSimpleOperatorValidation implements LoggedTest {
 
   private HelmParams opHelmParams = null;
   private V1ServiceAccount serviceAccount = null;
+  @NamespaceList
   private String opNamespace = null;
+  @NamespaceList
   private String domainNamespace1 = null;
+  @NamespaceList
```

Comment:
The above tagging won't be necessary if the decision is to use LogginUtil.getNSListIntersecting( )
Please add your review comments as to which way to use to get the namespace list.

Reply:
this is not what i was looking for, let's discuss at the daily meeting

Reply:
Implemented the new method of creating namespaces based on the Annotation at the class level. The TestWatcher creates it and passes it to the test instance for use. The list is can be used for log collection and can be used for cleanup once the test is done.